### ISSUE TITLE
Test Case 7: Ensure that consistent domain name is used for input URLs

### ISSUE THREAD

**Issue by StayCool21** [StayCool21/CS325_Project4#8]

**Comment by StayCool21** [comment]
We need to implement some sort of check to process each input URL and ensure they are all from **www.cbsnews.com** to ensure consistent output and formatting of the sanitized article.

**Comment by StayCool21** [comment]
Instead of directly editing and adding more code to `get_url_from_file`, I decided to make a new function to `mod1_funct_1.py` called `ensure_cbs_news_domain` in lines 40-49. I have it currently being called in `main` of `run.py` (line 28) to make sure that after all URLs are grabbed, they are from CBS News. I made sure to include the `https://` as I believe it's necessary for BeautifulSoup to parse. Let me know if it seems okay that we're halting the program if one domain does not match CBS News, or it seems better to skip that bad URL and continue on with execution. This should be ready for review.
